### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by developerwaleed**
# Vet clinic database: query multiple tables
### ScreenShots `schema.sql`:
image
image
image
image
image
image
### ScreenShots `data.sql`:
image
image
image
image
### ScreenShots `queries.sql`:
image
image
image
image
image
image
image

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -17,3 +17,41 @@ VALUES (' Charmander', '2020-02-08', 0, FALSE, -11),
...
+INSERT INTO species (name) 
+VALUES ('Pokemon'), ('Digimon');
+
+UPDATE animals 
+SET species_id = (SELECT id FROM species WHERE name = 'Digimon') 
+WHERE name LIKE '%mon';
+
+UPDATE animals 
+SET species_id = (SELECT id FROM species WHERE name = 'Digimon')
+WHERE name LIKE '%mon';
```
Comment (Ismailco) [inline data.sql]: - [x] You have duplicate queries here, kindly fix this. I think you want to target `Pokemon` here not `Digimon` and also pay attention to the `species_id` if that's your intention it should be where is null not like '%mon'

File: queries.sql [hunk 2]
```diff
@@ -63,4 +63,54 @@ FROM animals
...
+WHERE animals.species_id = (SELECT id FROM species WHERE name = 'Pokemon');
+
+SELECT full_name, name 
+FROM animals FULL 
+JOIN owners
+ON animals.owner_id = owners.id;
+
+SELECT COUNT(*), species.name AS Specie 
+FROM animals 
+RIGHT JOIN species
```
Comment (Ismailco) [inline queries.sql]: - [x] you should use LEFT Join here, kindly fix this

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @developerwaleed, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (developerwaleed) [reply data.sql]: Thankyou 😊

File: queries.sql (hunk 2 above)
Comment (developerwaleed) [reply queries.sql]: 👍

**Review by Deepakdanger — APPROVED** [review]
Hi @developerwaleed, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**